.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - run:
           name: "run tests"
           command: |
-            pushd sdk && npm ci && npm test && npm pack && popd
+            pushd sdk && npm ci --scope=@lightbend --registry=https://npm.cloudsmith.io/lightbend/akkaserverless && npm test && npm pack && popd
```

Comment:
I think this fixes the issue with CircleCI, although local installation won't work anymore.
Would it be possible to override the `install` & `ci` commands in the `script` section of the `package.json`?
Something like this:
```
  "scripts": {
    "print-version": "echo $npm_package_version",
    "watch-jsdoc": "npm-watch jsdoc",
    "jsdoc": "jsdoc -c jsdoc.json",
    "test": "mocha --recursive --unhandled-rejections=strict",
    "integration-test": "mocha integration-test",
    "preintegration-test": "bin/compile-descriptor.js test/example.proto --descriptor_set_out=integration-test/user-function.desc",
    "prepare": "bin/prepare.sh",
    "postinstall": "node bin/download-protoc.js",
    "prepublishOnly": "npm version --no-git-tag-version $(bin/version.sh)",
    "ci": "npm ci --registry https://npm.cloudsmith.io/lightbend/akkaserverless",
    "install": "npm install --registry https://npm.cloudsmith.io/lightbend/akkaserverless",
  }
```

I haven't tested it, but I think it should work.

Reply:
I changed the approach since the project `npmrc` configuration cannot be changed using command-line arguments:
https://docs.npmjs.com/cli/v7/configuring-npm/npmrc#per-project-config-file

Here in the CI and in the docs, we use the `userconfig` that can be more easily changed and tweaked:
https://docs.npmjs.com/cli/v7/using-npm/config#npmrc-files

The same applies during local development (ideally we should not publish to `@lightbend` from dev machines)

Reply:
@andreaTP I've checked out the branch and tried to do `npm install`, but go this error:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@lightbend%2ftsd-jsdoc - Not found
npm ERR! 404
npm ERR! 404  '@lightbend/tsd-jsdoc@2.5.1' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'sdk'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-06-22T14_55_08_201Z-debug.log
```

Reply:
@janory have you followed those instructions?
https://github.com/lightbend/akkaserverless-javascript-sdk/pull/45/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18